internal/broker/bind_create.go

```diff
@@ -112,19 +118,39 @@ func (b *BindEndpoint) Bind(ctx context.Context, instanceID, bindingID string, d
 	}
 
 	var kubeconfig string
+	binding := &internal.Binding{
+		ID:         bindingID,
+		InstanceID: instanceID,
+
+		CreatedAt: time.Now(),
+		UpdatedAt: time.Now(),
+
+		ExpirationSeconds: int64(expirationSeconds),
+	}
 	if parameters.ServiceAccount {
 		// get kubeconfig for the instance
 		kubeconfig, err = b.serviceAccountBindingManager.Create(ctx, instance, bindingID, expirationSeconds)
 		if err != nil {
 			message := fmt.Sprintf("failed to create kyma binding for service account using token request: %s", err)
 			return domain.Binding{}, apiresponses.NewFailureResponse(fmt.Errorf(message), http.StatusBadRequest, message)
 		}
+		binding.BindingType = internal.BINDING_TYPE_SERVICE_ACCOUNT
 	} else {
 		kubeconfig, err = b.gardenerBindingsManager.Create(ctx, instance, bindingID, expirationSeconds)
 		if err != nil {
 			message := fmt.Sprintf("failed to create kyma binding using adminkubeconfig gardener subresource: %s", err)
```

Comment:
```suggestion
			message := fmt.Sprintf("failed to create Kyma binding using adminkubeconfig gardener subresource: %s", err)
```

Reply:
Possibly we should create common naming policy - Kyma, Service Account, Gardener etc.